CMakeLists.txt

```diff
@@ -56,23 +55,12 @@ endif ()
 add_definitions(-DCMAKE)
 
 # Retrieve version from git into GIT_VERSION and creates VERSION.txt
-execute_process(COMMAND ${CMAKE_COMMAND}
-    -DGIT_BINARY=${GIT_BINARY}
-    -P ${CMAKE_SOURCE_DIR}/src/version.cmake
+execute_process(COMMAND python3 build-scripts/version.py
     ERROR_VARIABLE GIT_VERSION
```

Comment:
I think it's `OUTPUT_VARIABLE` now?

Reply:
The output from `version.py` is not needed anymore

Reply:
If it's not needed then we should remove `ERROR_VARIABLE GIT_VERSION` and `ERROR_STRIP_TRAILING_WHITESPACE` right?